README.md

```diff
@@ -452,6 +452,16 @@ ports can be done using the `consul_ports_*` variables.
   - Override with `CONSUL_TLS_SERVER_KEY` environment variable
 - Default value: `server.key`
 
+## `consul_encrypt_enable`
+
+- Enable Gossip Encryption
+- Default value: `true`
+
+## `consul_raw_key`
+
+- Set the encryption key; should be the same across a cluster
+- Default value: ``
```

Comment:
We might want to add a note that it will be auto generated or auto retrieved from existing servers if enabled but not provided.

Reply:
@groggemans Added based on the info in encrypt_gossip.yml; didn't realise it was doing that as my vagrant clusters encrypt-keys were blank ala `"encrypt": "",` which is how I ended up digging further down.